frontend/packages/ceph-storage-plugin/src/components/block-pool/create-block-pool.tsx

```diff
@@ -97,20 +97,6 @@ const CreateBlockPool: React.FC<CreateBlockPoolProps> = ({ match }) => {
 
   return (
     <>
-      <div className="co-create-operand__breadcrumbs">
-        <BreadCrumbs
-          breadcrumbs={[
-            {
-              name: params?.appName ? 'Openshift Container Storage' : 'Openshift Data Foundation',
-              path: blockPoolPageUrl,
-            },
-            {
-              name: t('ceph-storage-plugin~Create BlockPool'),
-              path: url,
-            },
-          ]}
-        />
-      </div>
       <div className="co-create-operand__header">
         <h1 className="co-create-operand__header-text">
```

Comment:
I intentionally didn't do that here because I was trying to limit the scope of the changes in the packages.  Per @spadgett, the packages are going to have to remove anything relying on CSS from `console` and use vanilla PatternFly.